clap_mangen/src/render.rs

```diff
@@ -33,7 +33,11 @@ pub(crate) fn synopsis(roff: &mut Roff, cmd: &clap::Command) {
     let name = cmd.get_bin_name().unwrap_or_else(|| cmd.get_name());
     let mut line = vec![bold(name), roman(" ")];
 
-    for opt in cmd.get_arguments().filter(|i| !i.is_hide_set()) {
+    let mut opts: Vec<_> = cmd.get_arguments().filter(|i| !i.is_hide_set()).collect();
+
+    opts.sort_by_key(|opt| opt.get_display_order());
```

Comment:
Should we be using the same sort keys as help?

https://github.com/clap-rs/clap/blob/4c039309b614ee4523c67a243afc38af11860de9/clap_builder/src/output/help_template.rs#L1063-L1088

Reply:
Whoops, looking at the wrong commit